system-addon/content-src/components/ContextMenu/ContextMenu.jsx

```diff
@@ -39,9 +44,11 @@ export class ContextMenuItem extends React.PureComponent {
     this.onKeyDown = this.onKeyDown.bind(this);
   }
 
-  onClick() {
-    this.props.hideContext();
-    this.props.option.onClick();
+  onClick(event) {
+    if (!this.props.option.disabled) {
```

Comment:
This change (`event` prop and `disabled` check) shouldn't be needed after the `pointer-events: none` as the click never goes to the option.